src/VisualStudio/IntegrationTest/New.IntegrationTests/AbstractWebApplicationTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Roslyn.VisualStudio.IntegrationTests;
+
+namespace Roslyn.VisualStudio.NewIntegrationTests
+{
+    public abstract class AbstractWebApplicationTest : AbstractEditorTest
+    {
+        protected static class GroupIds
+        {
+            public const string Server = "Microsoft.Web.Blazor.Server";
+            public const string Wasm = "Microsoft.Web.Blazor.Wasm";
```

Comment:
Ah, this is good to know. Thanks!

Reply:
I just copied from the Razor integration tests, so probably Ryan or Sam are to thank for this magic :)